packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -3,32 +3,81 @@ import ArrowRightIcon from 'coveo-styleguide/resources/icons/svg/arrow-right-rou
 import ArrowLeftIcon from 'coveo-styleguide/resources/icons/svg/arrow-left-rounded.svg';
 import {Button} from '../../common/button';
 
+type ArrowDirection = 'right' | 'left';
+
+/**
+ * @internal
+ * An 'atomic-segmented-facet-scrollable' wraps around one or several 'atomic-segmented-facet' to provide horizontal scrolling capabilities
+ * @part scrollableContainer - wrapper for the entire component including the horizontalScroll and the arrow buttons
+ * @part horizontalScroll - The scrollable container for the segmented facets
+ * @part left-arrow-box - The left arrow box containing both the left arrow button and the fade
+ * @part right-arrow-box - The right arrow box containing both the right arrow button and the fade
+ * @part arrow-button - The arrow button used to scroll left or right
+ * @part fade - The white to transparent gradient
+ */
+
 @Component({
   tag: 'atomic-segmented-facet-scrollable',
   styleUrl: 'atomic-segmented-facet-scrollable.pcss',
   shadow: true,
 })
 export class AtomicSegmentedFacetScrollable {
-  render() {
+  private horizontalScroll?: HTMLDivElement;
+
+  private slideHorizontally(direction: ArrowDirection) {
+    const container = this.horizontalScroll;
+    const pixelsToScroll = container ? container.clientWidth * 0.75 : 700;
+
+    if (container === null || !container) {
+      return;
+    }
+    if (direction === 'left') {
+      container.scrollLeft -= pixelsToScroll;
+    } else {
+      container.scrollLeft += pixelsToScroll;
+    }
+  }
+
+  private renderArrow(direction: ArrowDirection) {
+    const isLeft: boolean = direction === 'left';
     return (
-      <div class="flex h-9">
+      <div part={`${direction}-arrow-box`}>
         <Button
+          part="arrow-button"
           style="square-neutral"
-          class="flex shrink-0 basis-8 justify-center items-center rounded"
+          class={`flex shrink-0 basis-8 justify-center items-center rounded absolute z-10 w-10 top-0 bottom-0 ${
+            isLeft ? 'left-0' : 'right-0'
+          }`}
           ariaHidden="true"
+          onClick={() => this.slideHorizontally(direction)}
```

Comment:
maybe adding tabindex="-1" https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex is useful here too since tabbing with the keyboard normally moves the scroll, so accessing the arrow button with the keyboard isn't necessary

Reply:
I'm 50/50 on this, since keyboard users may see the buttons and expect to be able to reach them. Having `tabindex="-1"` would be more consistent with what we expect from screen readers though, so I guess it's a safer choice?